ccmlib/scylla_repository.py

```diff
@@ -122,3 +126,10 @@ def __get_dir():
     if not os.path.exists(repo):
         os.mkdir(repo)
     return repo
+
+
+def run_scylla_install_script(install_dir, target_dir):
+    subprocess.check_call(['bash', '-c', ''' sed 's|"$prefix|"$root/$prefix|' -i install.sh '''], cwd=install_dir)
+    subprocess.check_call(['bash', '-c', ''' {0}/install.sh --root {1}  --target  centos --disttype redhat --pkg server '''.format(install_dir, target_dir)], cwd=install_dir)
```

Comment:
nit: two spaces before and after `--target``
(and also you don't really need the extra space after the 3 single quotes and before the closing 3 quotes, do you?)